docs/google/calibration.md

```diff
@@ -0,0 +1,199 @@
+     metric_value = latest_calibration[metric_name][qubit_or_pair]
+     print(f'{qubit_or_pair} = {metric_value}')
+```
+
+Calibration metrics will also soon be available from the 
+[Google Cloud Platform Console](https://console.cloud.google.com).
+
+## Average, Pauli and Purity Error
+
+Several metrics below define average error, pauli error and purity error.
+The average error measures the error ε_a, averaged over all possible input
+states.  
+
+Pauli error defines decoherence of a single qubit in one of the Pauli channels
+X, Y, or Z.  If the errors are distributed in a uniform distribution across all
+three axes, the probability of applying an erroneous Pauli gate X, Y, or Z will
+be the Pauli error divided by three.  The Pauli error and average error are
+related by a multiplicative factor based on the number of qubits.
+
+See Table 1 on page 11 of the
+[https://arxiv.org/abs/1910.11333](Supplementry Information)
+document for a description and comparison between average error, Pauli error,
+and depolarization error.
+
+The [Purity](https://en.wikipedia.org/wiki/Purity_(quantum_mechanics)) of the
+quantum state is often defined as the trace of the density matrix of the
+state after the operations (RB or XEB) have been applied.  The purity error,
+or impurity, is one minus the purity, scaled to be from zero to one.  This is
+often referred to as the normalized
+[Linear Entropy](https://en.wikipedia.org/wiki/Linear_entropy).
+after the randomized benchmark circuit has been applied.  
+
+The purity error can be interpreted as a measure of the incoherent error,
+such as those caused by stochastic processes such as relaxation.  The average
+error can be interpreted as containing both this incoherent error as well as
+the coherent error resulting from improper control or calibration of the device.
+
+Note that, due to statistical fluctuations, it is possible that the purity
+error can exceed the average error by small amounts.
+
+For more about purity benchmarking, see Section 6.3 of this
+[thesis](https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf) .
+
+## Individual Metrics
+
+Each metric can be referenced by its key in the calibration object, e.g.
+```latest_calibration['single_qubit_idle_t1_micros']```.
+
+Note that the metric names below are subject to change without notice.
+
+### P_00 readout error
+
+*   Metric key: single_qubit_readout_p00_error
+
+The p00 readout error is the probability that a qubit is incorrectly measured as
+|1⟩ after being prepared in the |0⟩ state.  This is mostly a measure of the
+error in measurement (read out) of the qubit while in the ground state,
+though this probability also contains the error than the qubit was not reset
+into the |0⟩ ground state properly.  This is often called the SPAM (state 
+preparation and measurement) error.
+
+### P_11 readout error
+*   Metric key: single_qubit_readout_p11_error
+
+The P1 readout error is the probability that a qubit is incorrectly measured as
```

Comment:
Again, you have P1 where every where else it is P11. But also again, I think P1 makes more sense, because I don't know what the second 1 means. Perhaps you can explain?